src/SIL.XForge/Services/AuthService.cs

```diff
@@ -40,6 +40,19 @@ public bool ValidateWebhookCredentials(string username, string password)
 
     public Task<string> GetUserAsync(string authId) => CallApiAsync(HttpMethod.Get, $"users/{authId}");
 
+    public Task<string> GenerateTransparentUser(string name, string username, string password, string language)
+    {
+        var content = new JObject(
+            new JProperty("name", name),
+            new JProperty("username", username),
+            new JProperty("email", $"{username}@users.noreply.scriptureforge.org"),
```

Comment:
Have you checked various places where we might send a user an email (e.g. to reset their password) to see how we will handle this? We don't want to be sending emails into the void.